README.md

```
@@ -0,0 +1,25 @@
+## Installation
+
+To install Platzky Redirections, you can use pip:
+
+```sh
+pip install platzky-redirections
+```
+
+### Usage
+
+
+      "plugins": [
```

Markdownlint (0.37.0)

Expected: fenced; Actual: indented
Code block style

(MD046, code-block-style)